go.sum

```diff
@@ -1,3 +1,5 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
This is a little bit strange to see. Any idea where this is pulled in?

Reply:
It's being pulled in through Intel's VSP